python/test/unit/common/test_mpi.py

```diff
@@ -21,3 +27,55 @@ def test_mpi_comm_wrapper():
 
     assert isinstance(w1, MPI.Comm)
     assert isinstance(w2, MPI.Comm)
+
+
+def test_mpi_comm_wrapper_cppimport(tempdir):  # noqa: F811
+    """
+    Test MPICommWrapper <-> mpi4py.MPI.Comm conversion for code compiled with cppimport
+    """
+
+    cppimport = pytest.importorskip("cppimport")
```

Comment:
I am not sure how to change the circle CI configuration to make sure that cppimport is installed. Should I change the .yml file in .circleci, or the docker file?

I have tested them locally, but current CI run might skip the two new tests.

Reply:
It should be fine to install in the .circleci config for now.

Reply:
Best to put it here: https://github.com/FEniCS/dolfinx/blob/master/Dockerfile#L54

Reply:
Yes great, will put in Dockerfile. Do you want me to wait after #665 is merged to avoid conflicts? Even, if I were to change that line now, I would still need to wait till one of the two days of bi-weekly target to see any effect, right?

Reply:
I can manually trigger a build of the docker images, so go ahead.

Reply:
Merge conflict is trivial, don't worry about it.